komorebi/src/window_manager.rs

```diff
@@ -78,6 +78,7 @@ pub struct WindowManager {
     pub has_pending_raise_op: bool,
     pub pending_move_op: Option<(usize, usize, usize)>,
     pub already_moved_window_handles: Arc<Mutex<HashSet<isize>>>,
+    pub all_layouts: Vec<Layout>,
```

Comment:
If you implement the `cycle` method suggested I don't think this is necessary anymore

Reply:
Sorry, I forgot to remove this earlier. Fixed now.